extension/data.go

```diff
@@ -0,0 +1,301 @@
+package extension
+
+// OfferedExtensions contains the extensions our server handles
+// ourselves without redirecting to Google's component updater.
+var OfferedExtensions = Extensions{
+	{
+		ID:          "ldimlcelhnjgpjjemdjokpgeeikdinbm",
```

Comment:
i assume we just threw away the private keys that these IDs correspond to?

Reply:
They're in 1pwd.